### PR TITLE
Feature/sw 5 02/47 admin login/admin approve

### PR DESCRIPTION
**Pull request by jnordenso**
Added log in endpoint (using email/password)
Added role requests (all users that have no role yet, parent/teacher)
Added role approve (approving user request for role using ID)
User model updated

### CODE REVIEW

File: FoodplannerApi/Program.cs [hunk 1]
```diff
@@ -119,9 +119,9 @@
             var claimsIdentity = context.Principal.Identity as ClaimsIdentity;
 
             // Get the Status claim
-            var statusClaim = claimsIdentity?.FindFirst("Status")?.Value;
+            var statusClaim = claimsIdentity?.FindFirst("RoleApproved")?.Value;
```
Comment (alexrefshauge) [inline FoodplannerApi/Program.cs:122]: Kan vi undgå de her null-conditionals?

**Review by alexrefshauge — COMMENTED** [review]
Nice work, lige en enkelt kommentar 😊

File: FoodplannerApi/Program.cs (hunk 1 above)
Comment (alexrefshauge) [reply FoodplannerApi/Program.cs:122]: Jeg kan så også se at det ikke er jer der originalt lavede det, så i kan også bare resolve den 🤷‍♂️

**Review by JacobStokholms — APPROVED** [review]
Looks good

**Pull request merged**